buildconfig/CMake/FindJemalloc.cmake

```diff
@@ -0,0 +1,20 @@
+# Try to find the jemalloc library and include files
+#
+# JEMALLOC_INCLUDE_DIR where to find jemalloc.h, etc.
+# JEMALLOC_LIBRARIES libraries to link against
+# JEMALLOC_FOUND If false, do not try to use JEMALLOC
+
+find_path ( JEMALLOC_INCLUDE_DIR jemalloc.h
+            PATHS /usr/local/include/jemalloc
```

Comment:
I've just spotted an issue here. On RHEL/CentOS the header is under a `jemalloc` subdirectory in `/usr/include/jemalloc/jemalloc.h` so the finder fails on RHEL/CentOS and it's not included in the script despite the library being found.
As we only need the library anyway should be rename this to `FindJemallocLib.cmake` and just remove the `find_path` for the include? I've checked this and it works for CentOS.

Reply:
It works on Ubuntu too, so I changed that.